bin/node/cli/Cargo.toml

```diff
@@ -8,6 +8,9 @@ edition = "2018"
 license = "GPL-3.0"
 default-run = "substrate"
 
+[package.metadata.wasm-pack.profile.release]
+wasm-opt = false
```

Comment:
What's this line?

Reply:
`wasm-pack` seems to be shipping with a broken `wasm-opt`. When I build the wasm binary on either the build server or my machine, it maxes out _all_ the cores and doesn't seem to finish. Hopefully this'll get fixed soon.

Reply:
Do you have an issue number for what you're talking about? Let's link to it to explain the line so that people can remove it in the future

Reply:
Nothing specific to this issue, but `wasm-pack` seems to be having a lot of problems with `wasm-opt` on linux, https://github.com/rustwasm/wasm-pack/issues/782, https://github.com/rustwasm/wasm-pack/issues/781 and the recent changes don't seem to have improved things.

Reply:
Please at least add a comment explaining what this line is for.

Reply:
Done